src/rlx_mustache.erl

```diff
@@ -0,0 +1,230 @@
+%% The MIT License
+%%
+%% Copyright (c) 2009 Tom Preston-Werner <tom@mojombo.com>
+%%
+%% Permission is hereby granted, free of charge, to any person obtaining a copy
+%% of this software and associated documentation files (the "Software"), to deal
+%% in the Software without restriction, including without limitation the rights
+%% to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+%% copies of the Software, and to permit persons to whom the Software is
+%% furnished to do so, subject to the following conditions:
+%%
+%% The above copyright notice and this permission notice shall be included in
+%% all copies or substantial portions of the Software.
+%%
+%% THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+%% IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+%% FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+%% AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+%% LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+%% OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+%% THE SOFTWARE.
+
+%% See the README at http://github.com/mojombo/mustache.erl for additional
+%% documentation and usage examples.
+
+-module(rlx_mustache).  %% v0.1.0
+-author("Tom Preston-Werner").
+-export([compile/1, compile/2, render/1, render/2, render/3, get/2, get/3, escape/1, start/1]).
+
+-record(mstate, {mod = undefined,
+                 section_re = undefined,
+                 tag_re = undefined}).
+
+-define(MUSTACHE_STR, "rlx_mustache").
+
+compile(Body) when is_list(Body) ->
+  State = #mstate{},
+  CompiledTemplate = pre_compile(Body, State),
+  % io:format("~p~n~n", [CompiledTemplate]),
+  % io:format(CompiledTemplate ++ "~n", []),
```

Comment:
Are you intentionally leaving commented out stuff from the original author? Are we adopting this code? If so, we should at least make some kind of sanity pass over it as part of this PR to clean it up a tad.